src/main/java/com/microsoft/azure/helium/config/BuildConfig.java

```diff
@@ -12,10 +18,26 @@
 
     public String getBuildVersion(){
 
-        String buildName = buildProperties.getName();
-        String buildVersion = buildProperties.getVersion();
-        String buildTime =  String.valueOf(buildProperties.getTime().getEpochSecond());
-        System.out.println("buildName " + buildName +"_"+ "buildVersion "+ buildVersion + "_" + "buildTime "+ buildTime );
-        return buildVersion+"."+buildTime ;
+        Instant buildTime = buildProperties.getTime();
+
+        DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+                .withZone(ZoneId.systemDefault());
+
+        System.out.println(DATE_TIME_FORMATTER.format(buildTime));
+
+        //major.minor.MMdd.hhmm
+        String major = buildProperties.getVersion();
+        DateTimeFormatter DATE_TIME_FORMATTER1 = DateTimeFormatter.ofPattern("MMDD")
+                .withZone(ZoneId.systemDefault());
```

Comment:
does ZoneId.systemDefault() guarantee UTC?  What if the VM is running in a different TZ?

Reply:
Instant is an UTC hence I had used SystemDefault, however added with ZoneId.of("UTC") to be explicit as 
        DateTimeFormatter DATE_TIME_FORMATTER1 = DateTimeFormatter.ofPattern("MMDD").withZone( ZoneId.of("UTC"));